src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/presubmit/ApplicationUserRoleAppender.java

```diff
@@ -35,7 +35,8 @@ public boolean canHandle(
         requireNonNull(callback, "callback must not be null");
 
         return callbackStage == PreSubmitCallbackStage.ABOUT_TO_START
-            && (callback.getEvent() == Event.START_APPLICATION);
+            && (callback.getEvent() == Event.START_APPLICATION
```

Comment:
We don't need to re-write this field at this stage. Once it has been started viaLR, isLergalRep will always be true. Same for other users as well.

Reply:
I've reverted this, but I had to create a new aboutToStart handler to set a flag in ccd-definitions in order to know who's the user that is opening the case (an admin or a HO could be picking up a case from the case list which has isLegalRep set to Yes)

Reply:
In the handler you can find the loggedInUser as well.